ddl/callback.go

```diff
@@ -28,6 +33,8 @@ type Callback interface {
 	OnJobUpdated(job *model.Job)
 	// OnWatched is called after watching owner is completed.
 	OnWatched(ctx context.Context)
+	// OnGetInfoSchema is called in getting information schema. It's used for testing now.
+	OnGetInfoSchema(ctx sessionctx.Context, fn GetInfoSchema) infoschema.InfoSchema
```

Comment:
OnGetInfoSchema is a hook which is called in the function ddl.GetInfoSchema(). It is used in the tests.